packages/dx-react-grid/docs/guides/grouping.md

```diff
@@ -1,41 +1,45 @@
 # React Grid Data Grouping
 
-## Overview
+The Grid component supports grouping data by one or several column values. Use the corresponding plugins to manage the grouping state and group data programmatically or via the UI (Group Panel and column headers).
 
-The grouping feature allows you to display data grouped by one or several column values. We also provide plugins that allow end-users to group grid data using the Grid's UI (group panel or column headers).
+## Related Plugins
 
-## Plugin List
+The following plugins implement grouping features:
 
-There are several plugins that implement grouping features:
-- [GroupingState](../reference/grouping-state.md)
-- [LocalGrouping](../reference/local-grouping.md)
-- [TableGroupRow](../reference/table-group-row.md)
-- [TableHeaderRow](../reference/table-header-row.md)
-- [GroupingPanel](../reference/grouping-panel.md)
+- [GroupingState](../reference/grouping-state.md) - controls the grouping state  
+- [LocalGrouping](../reference/local-grouping.md) - implements the grouping logic  
+- [TableGroupRow](../reference/table-group-row.md) - renders group rows  
+- [TableHeaderRow](../reference/table-header-row.md) - renders the header row and implements column dragging  
+- [GroupingPanel](../reference/grouping-panel.md) - renders the Group Panel
 
 Note that [plugin order](../README.md#plugin-order) is important.
 
 ## Basic setup
 
-Use the `GroupingState`, `LocalGrouping` and `TableGroupRow` plugins to set up a simple static Grid grouping.
+Use the `GroupingState`, `LocalGrouping` and `TableGroupRow` plugins to set up a Grid with simple static grouping. 
 
-In the following example, the controlled mode is enabled using the `grouping` property of the `GroupingState` plugin. You do not have to handle the `onGroupingChange` event because an end-user can not change grouping state via the UI.
+In the following example, the grouping options are specified using the `GroupingState` plugin's `grouping` property, which is typical for the controlled mode. However, the `onGroupingChange` event handler is not specified because the grouping option is not supposed to be changed internally due to the grouping UI disability.
```

Comment:
"because the grouping option is not supposed to be changed internally due to the grouping UI disability." This part does not make sense.